src/vm/ast/mod.rs

```diff
@@ -0,0 +1,28 @@
+pub mod parser;
+pub mod expression_identifier;
+pub mod sugar_expander;
+pub mod types;
+pub mod errors;
+use vm::errors::Error;
+
+use vm::representations::{SymbolicExpression};
+use vm::types::QualifiedContractIdentifier;
+
+use self::types::{ContractAST, BuildASTPass};
+use self::errors::{ParseResult};
+use self::expression_identifier::ExpressionIdentifier;
+use self::sugar_expander::SugarExpander;
+
+/// Legacy function
+pub fn parse(contract_identifier: &QualifiedContractIdentifier,source_code: &str) -> Result<Vec<SymbolicExpression>, Error> {
```

Comment:
Does this need to take a QualifiedContractIdentifier as an argument? Or does it make more sense to just accept the `issuer` ?

Reply:
It's fine for this to accept the QualifiedContractIdentifier, I think. This only gets invoked for contract launches, which means both a contract name and an issuer are available.